scapy/layers/l2.py

```diff
@@ -539,11 +539,12 @@ class EAP_MD5(EAP):
         ByteEnumField("code", 1, eap_codes),
         ByteField("id", 0),
         FieldLenField("len", None, fmt="H", length_of="optional_name",
-                      adjust=lambda p, x: x + p.value_size + 6),
+                adjust=lambda p, x: (x + p.value_size + 6) if p.value_size is not None else 6),
```

Comment:
Please keep the correct alignment, it seems broken now (at least in Github).
Can you reverse the test (`if p.value_size is None`)?
Also, do you want `6` when `p.value_size is None` or `x + 6`?
Maybe, in that case, something like `lambda p, x: x + 6 + (0 if p.value_size is None else p.value_size)` would be easier to read. Or even easier: `lambda p, x: x + 6 + (p.value_size or 0)`.